### PR TITLE
Fix too verbose log on tests

### PR DESCRIPTION
**Pull request by thebevrishot**
closes https://github.com/zcoinofficial/ztm/issues/19

### CODE REVIEW

File: src/Ztm.Zcoin.NBitcoin.Tests/RpcTests.cs [hunk 1]
```diff
@@ -17,7 +17,9 @@ public class RpcTests : IDisposable
         public RpcTests()
         {
             this.builder = NodeBuilderFactory.CreateNodeBuilder(GetType().Name);
-            this.node = this.builder.CreateNode(true);
+            this.node = this.builder.CreateNode(false);
+            this.node.ConfigParameters.Add("printtoconsole", "0");
```
Comment (ultimaweapon) [inline src/Ztm.Zcoin.NBitcoin.Tests/RpcTests.cs]: We should apply this in `NodeBuilderFactory.CreateNodeBuilder` instead so we don't need to do this everywhere.

File: src/Ztm.Zcoin.NBitcoin.Tests/RpcTests.cs [hunk 2]
```diff
@@ -17,7 +17,9 @@ public class RpcTests : IDisposable
         public RpcTests()
         {
             this.builder = NodeBuilderFactory.CreateNodeBuilder(GetType().Name);
-            this.node = this.builder.CreateNode(true);
+            this.node = this.builder.CreateNode(false);
```
Comment (ultimaweapon) [inline src/Ztm.Zcoin.NBitcoin.Tests/RpcTests.cs]: If we go with the way as I commented we don't need to change this line.

File: src/Ztm.Zcoin.NBitcoin.Tests/RpcTests.cs [hunk 3]
```diff
@@ -17,7 +17,9 @@ public class RpcTests : IDisposable
...
+            this.node.ConfigParameters.Add("printtoconsole", "0");
+            this.node.StartAsync().Wait();
```
Comment (ultimaweapon) [inline src/Ztm.Zcoin.NBitcoin.Tests/RpcTests.cs]: Same here.

File: src/Ztm.Zcoin.Rpc.Tests/ZcoinRpcClientFactoryTests.cs [hunk 4]
```diff
@@ -33,7 +33,9 @@ public async Task CreateRpcClientAsync_WithCorrectParameters_ShouldSuccess()
         {
             using (var nodeBuilder = NodeBuilderFactory.CreateNodeBuilder(GetType().Name))
             {
-                var node = nodeBuilder.CreateNode(true);
+                var node = nodeBuilder.CreateNode(false);
```
Comment (ultimaweapon) [inline src/Ztm.Zcoin.Rpc.Tests/ZcoinRpcClientFactoryTests.cs]: Same here.

File: src/Ztm.Zcoin.Rpc.Tests/ZcoinRpcClientFactoryTests.cs [hunk 5]
```diff
@@ -33,7 +33,9 @@ public async Task CreateRpcClientAsync_WithCorrectParameters_ShouldSuccess()
...
+                node.ConfigParameters.Add("printtoconsole", "0");
```
Comment (ultimaweapon) [inline src/Ztm.Zcoin.Rpc.Tests/ZcoinRpcClientFactoryTests.cs]: Same here.

File: src/Ztm.Zcoin.Rpc.Tests/ZcoinRpcClientFactoryTests.cs [hunk 6]
```diff
@@ -33,7 +33,9 @@ public async Task CreateRpcClientAsync_WithCorrectParameters_ShouldSuccess()
...
+                node.StartAsync().Wait();
```
Comment (ultimaweapon) [inline src/Ztm.Zcoin.Rpc.Tests/ZcoinRpcClientFactoryTests.cs]: Same here.

File: src/Ztm.Zcoin.Rpc.Tests/ZcoinRpcClientTests.cs [hunk 7]
```diff
@@ -26,6 +26,7 @@ public ZcoinRpcClientTests()
                 this.nodes.ConfigParameters.Add("autocommit", "0");
                 this.nodes.ConfigParameters.Add("dandelion", "0");
                 this.nodes.ConfigParameters.Add("exodus", "1");
+                this.nodes.ConfigParameters.Add("printtoconsole", "0");
```
Comment (ultimaweapon) [inline src/Ztm.Zcoin.Rpc.Tests/ZcoinRpcClientTests.cs]: Same here.

File: src/Ztm.Zcoin.NBitcoin.Tests/RpcTests.cs (hunk 1 above)
Comment (thebevrishot) [reply src/Ztm.Zcoin.NBitcoin.Tests/RpcTests.cs]: agree

**Pull request merged**